flask_sse.py

```diff
@@ -96,7 +108,7 @@ class ServerSentEventsBlueprint(Blueprint):
     A :class:`flask.Blueprint` subclass that knows how to publish, subscribe to,
     and stream server-sent events.
     """
-    @property
+    @lazy_property
```

Comment:
Please use the `cached_property` decorator from Werkzeug, rather than implementing a copy yourself. https://github.com/pallets/werkzeug/blob/master/werkzeug/utils.py#L35